docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: '3.8'
+
+services:
+  dev_app:
+    build:
+      context: .
+      target: development
+    volumes:
+      - .:/app
+    ports:
+      - "5000:5000"
+    environment:
+      - FLASK_APP=todo_app/app.py
+      - FLASK_ENV=development
+    command: poetry run flask run --host=0.0.0.0 --reload
```

The command here I think could be dropped in favour of the default entrypoint, and we could pass through all the `.env` file as we do in the run commands rather than relying on the volume to copy the `.env` file directly if we wanted:
```suggestion
    env_file:
      - .env

```